src/main/flight/pid.c

```diff
@@ -639,7 +640,7 @@ static void rotateVector(float v[XYZ_AXIS_COUNT], const float rotation[XYZ_AXIS_
     }
 }
 
-STATIC_UNIT_TESTED void rotateItermAndAxisError(void)
+STATIC_UNIT_TESTED FAST_CODE_NOINLINE void rotateItermAndAxisError(void)
```

Comment:
Originally it was because I ran into issues with overflowing F7x fast ram.  I think in master it is OK now so, yes I can remove this, probably.

Reply:
Sorry, I misunderstood what does `FAST_CODE_NOINLINE` mean.

Reply:
I am not expert on this, but I understand that it is a macro which stops the code block going into the fast flash space for F7x and similar hardware.  We were hitting the limit of this space around the time I was developing the PR, so I chose some code blocks that are not used much and applied the macro, otherwise Github wouldn't pass the unit test since it wouldn't fit on those microcontrollers.  Since that time, some other PR's must have freed up some fast ram space on F7's, so I don't need to make this change now.